catalog/nginx-web.conf

```diff
@@ -25,4 +25,9 @@ server {
 		add_header X-Frame-Options DENY;
 		try_files $uri /index.html =404;
 	}
+
+    location = /healthcheck {
```

Comment:
Just use `/` for healthchecking in the CFN template?

Reply:
I tried that and it didn't work. I'm not sure exactly why.